test/org/opensolaris/opengrok/index/IndexDatabaseTest.java

```diff
@@ -77,27 +90,85 @@ public static void tearDownClass() throws Exception {
     public void testGetDefinitions() throws Exception {
         // Test that we can get definitions for one of the files in the
         // repository.
-        File f1 = new File(repository.getSourceRoot() + "/c/foobar.c");
+        File f1 = new File(repository.getSourceRoot() + "/git/main.c");
         Definitions defs1 = IndexDatabase.getDefinitions(f1);
         assertNotNull(defs1);
-        assertTrue(defs1.hasSymbol("foobar"));
-        assertTrue(defs1.hasSymbol("a"));
+        assertTrue(defs1.hasSymbol("main"));
+        assertTrue(defs1.hasSymbol("argv"));
         assertFalse(defs1.hasSymbol("b"));
-        assertTrue(defs1.hasDefinitionAt("foobar", 1, new String[1]));
+        assertTrue(defs1.hasDefinitionAt("main", 3, new String[1]));
 
         //same for windows delimiters
-        f1 = new File(repository.getSourceRoot() + "\\c\\foobar.c");
+        f1 = new File(repository.getSourceRoot() + "\\git\\main.c");
         defs1 = IndexDatabase.getDefinitions(f1);
         assertNotNull(defs1);
-        assertTrue(defs1.hasSymbol("foobar"));
-        assertTrue(defs1.hasSymbol("a"));
+        assertTrue(defs1.hasSymbol("main"));
+        assertTrue(defs1.hasSymbol("argv"));
         assertFalse(defs1.hasSymbol("b"));
-        assertTrue(defs1.hasDefinitionAt("foobar", 1, new String[1]));
+        assertTrue(defs1.hasDefinitionAt("main", 3, new String[1]));
 
         // Test that we get null back if we request definitions for a file
         // that's not in the repository.
-        File f2 = new File(repository.getSourceRoot() + "/c/foobar.d");
+        File f2 = new File(repository.getSourceRoot() + "/git/foobar.d");
         Definitions defs2 = IndexDatabase.getDefinitions(f2);
         assertNull(defs2);
     }
+
+    private void checkDataExistence(String fileName, boolean shouldExist) {
+        RuntimeEnvironment env = RuntimeEnvironment.getInstance();
+
+        for (String dirName : new String[]{"historycache", IndexDatabase.XREF_DIR}) {
```

Comment:
I'd rather keep it not exposed. Changing the field to `protected` does not help.